src/WebJobs.Script.WebHost/Security/KeyManagement/BlobStorageSecretsRepository.cs

```diff
@@ -19,6 +21,7 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost
     /// </summary>
     public class BlobStorageSecretsRepository : BaseSecretsRepository
     {
+        private const string BlobArchivedName = "BlobArchived";
```

Comment:
```suggestion
        private readonly string _blobArchivedName = "BlobArchived";
```
Consider following the conventions in the file.

Reply:
Fixed.